deb-get: add page (#8172)
tldr-pages/tldr: pages/common/deb-get.md

# deb-get

> > The apt-get functionality for .debs published in 3rd party repositories or via direct download. It works on Ubuntu and derivative distributions.

Reviewer: ```suggestion
> `apt-get` functionality for `.deb` packages published in third party repositories or via direct download.
> It works on Debian, Ubuntu, and derivative distributions.
```

# deb-get

> `apt-get` functionality for `.deb` packages published in third party repositories or via direct download.

> > Works with Linux distributions which use apt-get.

Reviewer: ```suggestion
> Works with Linux distributions which use `apt-get`.
```

> > It works on Debian, Ubuntu, and derivative distributions.

Reviewer 2: ```suggestion
> Works with Linux distributions with apt-get.
```

> More information: <https://github.com/wimpysworld/deb-get>.

> - Update the list of available packages and versions (it's recommended to run this before other commands):

Reviewer 2: ```suggestion
- Update the list of available packages and versions:
```

`sudo deb-get update`

- Search for a given package:

`sudo deb-get search {{package}}`

> - Show information for a package:

Reviewer 3: Not entirely sure, but I think `for` can't be used here.
```suggestion
- Show information about a package:
```

`sudo deb-get show {{package}}`

- Install a package, or update it to the latest available version:

`sudo deb-get install {{package}}`

- Remove a package (using `purge` instead also removes its configuration files):

`sudo deb-get remove {{package}}`

- Upgrade all installed packages to their newest available versions:

`sudo deb-get upgrade`

> - List all packages:

Reviewer 3: ```suggestion
- List all available packages:
```
Otherwise it may be interpreted as `installed packages`.

> `sudo deb-get upgrade`

Reviewer 2: Could we move this command to below the update command please

Reviewer 3: @CleanMachine1 then it should also be moved in the `apt-get` page. It has the same command order.
